salt/_auth/kubernetes_rbac.py

```diff
@@ -0,0 +1,69 @@
+import base64
+import logging
+
+log = logging.getLogger(__name__)
+
+try:
+    import kubernetes.client
+    import kubernetes.config
+    HAS_DEPS = True
+except ImportError:
+    HAS_DEPS = False
+
+__virtualname__ = 'kubernetes_rbac'
+
+def __virtual__():
+    if not HAS_DEPS:
+        return False, 'Missing dependencies'
```

Comment:
We should mention which deps are missing in the error message, otherwise it's not really useful.

Reply:
Given https://github.com/scality/metalk8s/pull/760/files#r273824915 this will be removed.